tests/cpp/test_multidevice_lower_communication.cpp

```diff
@@ -25,15 +25,17 @@ using testing::Property;
 namespace {
 void assertIsCompiledToHostIrContainer(
     const FusionExecutorCache& executor_cache) {
-  FusionKernelRuntime* runtime = executor_cache.getMostRecentKernelRuntime();
-  EXPECT_EQ(runtime->executors().size(), 1);
-  EXPECT_THAT(
-      runtime->executors(),
-      Each(Pointer(Property(
-          "is a HostIrExecutor",
-          &ExecutorAbstract::isA<HostIrExecutor>,
-          IsTrue()))))
-      << "failed to compile to a HostIrContainer with Communications";
+  if (!isOptionEnabled(EnableOption::HostIrLowering)) {
```

Comment:
Sure -- this is a valid change. But can you come up with a check for host IR lowering? For example, check the top level instructions in a HostIrContainer has communications?